Client/Frontend/Browser/Search/SearchLoader.swift

```diff
@@ -73,6 +73,13 @@ class _SearchLoader<UnusedA, UnusedB>: Loader<[Site], SearchViewController> {
                 
                 // First, see if the query matches any URLs from the user's search history.
                 self.load(result)
+                
+                // If the new search string is not longer than the previous
+                // we don't need to find an autocomplete suggestion.
+                if oldValue.count > self.query.count {
```

Comment:
Will this not fail when I select the entire query and start typing again. ie on search bar type a query, then select all and type a letter for new query? (Of course this will only fail for the first replacing character, post the first it will start working again).

Reply:
Yes, it will not autofill after first character, do you think it's worth fixing? any smart solution for this?

Reply:
perhaps it's even good, as query suggestion based on 1 character isn't that useful

Reply:
Yeah, I don't see any problems with this. I am ok just wanted to bring it to notice.

Reply:
FYI, maybe you can add another check like `if oldValue.count > self.query.count && oldValue.hasPrefix(self.query) {` or something on that line

Reply:
either is fine